README.md

```
@@ -56,16 +57,16 @@ import (
 	// Create a client with configuration from environment variables.
 	client := smsgateway.NewClient(smsgateway.Config{
-		User:     os.Getenv("GATEWAY_USER"),
-		Password: os.Getenv("GATEWAY_PASSWORD"),
+		User:     os.Getenv("ASG_USERNAME"),
+		Password: os.Getenv("ASG_PASSWORD"),
 	})
 
 	// Create a message to send.
 	message := smsgateway.Message{
-		Message: "Your SMS message text here",
+		Message: "Hello, doctors!",
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)